eclass/waf-utils.eclass

```diff
@@ -19,7 +19,7 @@
 inherit multilib toolchain-funcs multiprocessing
 
 case ${EAPI:-0} in
-	4|5|6) EXPORT_FUNCTIONS src_configure src_compile src_install ;;
+	4|5|6|7) EXPORT_FUNCTIONS src_configure src_compile src_install ;;
```

Comment:
The `EXPORT_FUNCTIONS` assignment is the same for all supported EAPIs, so it should be moved out of the `case` statement (i.e., leaving an empty clause).